torchtitan/experiments/torchcomms/README.md

```diff
@@ -1,25 +1,32 @@
 ## TorchTitan & TorchComms Composability Testing
 
-#### Overview
+### Overview
 
-This folder provides a framework for composability testing with TorchComms and distributed training in TorchTitan. The goal is to enable flexible experimentation with distributed communication primitives and parallelism strategies in PyTorch.
-TODO: add more explanation once the torchcomm goes public.
----
-#### Example
+This folder provides a framework for composability testing with TorchComms and distributed training in TorchTitan. It enables flexible experimentation with distributed communication primitives and various parallelism strategies in PyTorch.
+
+> **TODO:** Additional documentation will be provided once TorchComms is publicly released.
+
+### Quick Start
+
+The following command uses Llama 3 as an example, but should work with all models:
 
-The command below uses Llama 3 as an example, but should work on all models.
 ```bash
 TEST_BACKEND=nccl TRAIN_FILE=torchtitan.experiments.torchcomms.train CONFIG_FILE="./torchtitan/models/llama3/train_configs/debug_model.toml" ./run_train.sh
 ```
----
-### Available Features
-- **Distributed Training Utilities**
-  - Training with `torchcomms.new_comm`
-  - Device mesh initialization with `torchcomms.init_device_mesh`
-- **Composability Testing**
-  - Integration and testing with `fully_shard` (FSDP)
----
-### To Be Added
-- Integration and testing with additional parallelism strategies (e.g., tensor, pipeline, context parallelism) other than fully_shard
-- Integration and testing with torch.compile
----
+
+### Features
+
+#### Distributed Training Utilities
+- Custom communicator initialization via `torchcomms.new_comm`
+- Device mesh setup using `torchcomms.init_device_mesh`
+
+#### Parallelism Support
+Fully integrated and tested with:
```

Comment:
We have not tested the convergence yet? Right now let's just say locally tested? And in TODO, let's add a N-D parallelism E2E perf and convergence test?